<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BillWidget</class>
 <widget class="QWidget" name="BillWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BillWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>录入</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="3">
             <widget class="QCheckBox" name="relics">
              <property name="text">
               <string/>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>成本：</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="ranking">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLineEdit" name="costPrice">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="5">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>个人利润：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QLabel" name="label_6">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>总利润：</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>排名：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDateEdit" name="dateEdit">
              <property name="dateTime">
               <datetime>
                <hour>0</hour>
                <minute>0</minute>
                <second>0</second>
                <year>2016</year>
                <month>2</month>
                <day>14</day>
               </datetime>
              </property>
              <property name="displayFormat">
               <string>yyyy-MM-dd</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>日期：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="6">
             <widget class="QLineEdit" name="totalProfit">
              <property name="readOnly">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="6">
             <widget class="QLineEdit" name="personalProfit">
              <property name="readOnly">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>是否真迹：</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QLineEdit" name="totalTurnover">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>编号：</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="name">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="billCount">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>单数：</string>
              </property>
             </widget>
            </item>
            <item row="3" column="6">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="importBtn">
                <property name="text">
                 <string>导入数据库</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="2" column="6">
             <widget class="QLineEdit" name="fundsTransfer">
              <property name="readOnly">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>总成交额：</string>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="calculateBtn">
                <property name="text">
                 <string>计算</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="2" column="5">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>转账资金：</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>数据</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="TableView" name="tableView">
            <property name="minimumSize">
             <size>
              <width>1050</width>
              <height>300</height>
             </size>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustIgnored</enum>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QToolButton" name="deleteBtn">
            <property name="text">
             <string>删除此行</string>
            </property>
            <property name="icon">
             <iconset resource="account.qrc">
              <normaloff>:/Account/Resources/Entypo_e729(0)_64.png</normaloff>:/Account/Resources/Entypo_e729(0)_64.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>tableview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>dateEdit</tabstop>
  <tabstop>ranking</tabstop>
  <tabstop>billCount</tabstop>
  <tabstop>name</tabstop>
  <tabstop>relics</tabstop>
  <tabstop>costPrice</tabstop>
  <tabstop>totalTurnover</tabstop>
  <tabstop>calculateBtn</tabstop>
  <tabstop>totalProfit</tabstop>
  <tabstop>personalProfit</tabstop>
  <tabstop>fundsTransfer</tabstop>
  <tabstop>importBtn</tabstop>
  <tabstop>deleteBtn</tabstop>
 </tabstops>
 <resources>
  <include location="account.qrc"/>
 </resources>
 <connections/>
</ui>
